<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="2">
    <widget class="QPushButton" name="previousPushButton">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find</string>
     </property>
     <property name="buddy">
      <cstring>findLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="replaceLabel">
     <property name="text">
      <string>Replace</string>
     </property>
     <property name="buddy">
      <cstring>replaceLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="findLineEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="replaceLineEdit"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="nextPushButton">
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findLineEdit</tabstop>
  <tabstop>previousPushButton</tabstop>
  <tabstop>nextPushButton</tabstop>
  <tabstop>replaceLineEdit</tabstop>
  <tabstop>toolButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>released()</signal>
   <receiver>SearchWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
